cri-o.yml

```diff
@@ -42,6 +42,7 @@
         - build-essential
         - curl
         - gawk
+        - git
```

Comment:
Well there are tasks to clone git repositories and my Ubuntu installation didn't had git by default.